### ISSUE TITLE
Capturing only works only works if protocol is exposed as false intially under Firefox

### ISSUE THREAD

**Issue by Thaodan** [alphapapa/org-protocol-capture-html#49]
Hey, I've noticed that registering the protocol only works for me with Firefox (104) if I first expose the org-protocol with false and then click a link with the org-protocol like this:
`data:text/html,<a href="org-protocol://resolve?domain=Bold">Link</a>`

Source:
https://www.reddit.com/r/firefox/comments/6p6470/comment/dpap5ok/?context=3

**Comment by alphapapa** [comment]
Hi,

I'm not sure I completely understand.  Are you proposing an update to the documentation?

**Comment by Thaodan** [comment]
Yes. While I'm not exactly sure why this has happened but for my case
this was needed to have the protocol recognized.

**Comment by alphapapa** [comment]
Ok, feel free to send a pull request for the readme.  Please mention this in a separate paragraph so it's clear that we're not sure whether it's always necessary.

**Comment by stefan2904** [comment]
Since I needed to configure a fresh Firefox, I just tried to reproduce this. After setting `network.protocol-handler.expose.org-protocol` to false (and no other org-protocol related flags), and using your html, I end up with this dialog, where clicking on the right button does not open any further dialogs and nothing is invoked:

![image](https://user-images.githubusercontent.com/527767/232505990-760d3849-56b2-4ad1-9b9a-2c96016a781a.png)

Than I noticed that firefox automatically figured out which desktop entry to use (probably using xdg): 

![image](https://user-images.githubusercontent.com/527767/232506661-29670b5d-4d4f-4db6-aca7-ef277cf3018c.png)

But opening any org-protocol link still does not invoke it. (manually invoking `xdg-open "org-protocol://foo" ` works.)
